### ISSUE TITLE
Discussion - PayloadTooLargeError: request entity too large

### ISSUE THREAD

**Issue by huygn** [netlify/netlify-lambda#40]
Got this error when testing my function locally by sending a ~160kb POST request. This error doesn't happen when deployed on Netlify functions though.

This is due to express server with it's default limit and could be altered via [middlewares](https://stackoverflow.com/a/19965089/4328963). Could we allow express middlewares from `netlify-lambda` users via `netlify-lambda-serve.js` or similar mechanism for flexibilities?

**Comment by willdavidow** [comment]
You can solve this issue by setting a file size limit for the express instance's body parser:
I had the same thing happening, and just fixed it - here's my `netlify-lambda` relevant setup:

```
var bodyParser = require('body-parser');
var proxy = require('http-proxy-middleware');

module.exports = {
  developMiddleware: app => {
    console.log('Netlify Lambda Middleware Bootstrap');
    app.use(
      bodyParser.urlencoded({
        extended: true,
        limit: '50mb',
        parameterLimit: 1000000
      })
    );

    app.use(
      '/.netlify/functions',
      proxy({
        target: 'http://localhost:4567',
        pathRewrite: {
          '/.netlify/functions': '',
        },
      })
    );
  },
```

**Comment by sw-yx** [comment]
sounds reasonable, but will not have bandwidth to work on this for the time being - keeping this issue open until we can address this

**Comment by 8eecf0d2** [comment]
_Afaict_ this is a non issue, adding a `netlify-lambda-serve.js` feature would not align at all with the Netlify Functions service and only fragment development / production further.

The original body size limit issue was resolved with #31.

**Comment by sw-yx** [comment]
thanks for the second opinion Brod. think im just gonna close this for now.
